pkgs/by-name/li/libvdpau-va-gl/package.nix

```
@@ -26,6 +27,25 @@ stdenv.mkDerivation rec {
     sha256 = "0asndybfv8xb0fx73sjjw5kydqrahqkm6n04lh589pbf18s5qlld";
   };
 
+  patches = [
+    # cmake-4 compatibility
+    (fetchpatch {
+      name = "cmake-4-1.patch";
+      url = "https://github.com/i-rinat/libvdpau-va-gl/commit/30c8ac91f3aa2843f7dc1c1d167e09fad447fd91.patch";
```

```suggestion
      url = "https://github.com/i-rinat/libvdpau-va-gl/commit/30c8ac91f3aa2843f7dc1c1d167e09fad447fd91.patch?full_index=1";
```

I believe github commit fetches should get a `full_index` for reproducibility reasons.